### PR TITLE
update

### PR DESCRIPTION
**Pull request by AbusiddikAntor**
update in pdf date

### CODE REVIEW

File: frontend/src/utils/timeZone.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { addDays, isBefore, set } from "date-fns";
 import { getTodayFormateDate } from "./utils";
-import moment from "moment";
+// import moment from "moment";
```
Comment (devBasar) [inline frontend/src/utils/timeZone.js:3]: please download moment js and remove the comment

**Review by devBasar — COMMENTED** [review]
please add moment js in utils/timeZone.js file

**Review by devBasar — APPROVED** [review]
Moment js import successfully and there have some changes in yarn.lock file but we can merge it.

File: frontend/src/utils/timeZone.js (hunk 1 above)
Comment (AbusiddikAntor) [reply frontend/src/utils/timeZone.js]: solve this moment js

**Pull request merged**